LOCALIZE.md

```diff
@@ -93,7 +93,7 @@ For each `word`, an optional _.png_ image exists. In Swahili, the words _mbu, ki
 
```

Comment:
Can you update to mention about the process to update the instructions?

Reply:
Already done in section 5 `Component audio`